### ISSUE TITLE
skip_list skips everything

### ISSUE THREAD

**Issue by p7kdev** [ansible/ansible-lint#4040]
##### Summary

Hi,

I would like to skip a specific tag completely (`var-naming[no-role-prefix]`). I have created the following `.ansible-lint` configuration

```yaml
---
skip_list:
  - var-naming[no-role-prefix]
```

But now no check is executed at all. Everything seems ok.

If i run ansible-lint without a `.ansible-lint` configuration file, i will get the following ouput

```
                       Rule Violation Summary                        
 count tag                        profile rule associated tags       
   307 var-naming[no-role-prefix] basic   idiom                      
     1 yaml[empty-lines]          basic   formatting, yaml           
     3 yaml[indentation]          basic   formatting, yaml           
     1 yaml[key-duplicates]       basic   formatting, yaml           
     1 risky-shell-pipe           safety  command-shell              
     7 no-changed-when            shared  command-shell, idempotency 

Failed: 320 failure(s), 0 warning(s) on 75 files. Last profile that met the validation criteria was 'min'.
```

If I skip one tag now, I would have expected the others to still appear. But with the above config I now get the following

```
Passed: 0 failure(s), 0 warning(s) on 1 files. Last profile that met the validation criteria was 'production'.
```

But I would still like to have the other messages because I would like to fix them. Just not the one which i want to skip.

Background. I have a collection `namespace.zabbix` and in it several roles (agent, server, proxy, ...). Now my variables have the structure `zabbix_ROLE_variable` (e.g. `zabbix_agent_port`), but ansible-lint complains about it. (That's ok, it's defined that way). But surely this one check can be ignored somehow.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint 6.22.1 using ansible-core:2.16.3 ansible-compat:4.1.11 ruamel-yaml:0.17.40 ruamel-yaml-clib:0.2.8
```

- ansible installation method: OS package (Archlinux)
- ansible-lint installation method: OS package (Archlinux)

##### STEPS TO REPRODUCE

Add a simple `.ansible-lint` configuration with `skip_list`

```yaml
---
skip_list:
  - var-naming[no-role-prefix]
```

##### Desired Behavior

Only the one check is ignored

##### Actual Behavior

Everything is ignored

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by ssbarnea** [comment]
Please link to a minimal repository that reproduced the bug.

**Comment by Durf1985** [comment]
I have same problems
